.github/workflows/ps-ce-sdk-release.yaml

```diff
@@ -0,0 +1,44 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+name: Release
+
+on:
+  create:
+    tags:
+      - v*
+
```

Comment:
Please also include `workflow_dispatch:` which allows us to manually trigger a release from CLI (example: https://github.com/embano1/ci-demo-app/blob/main/.github/workflows/release.yaml)